.github/workflows/tests.yml

```diff
@@ -27,9 +33,8 @@ jobs:
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.6
-    - name: Install dependencies
-      run: bundle install
+        ruby-version: ${{ matrix.ruby-version }}
```

Comment:
> I didn't know if it made sense to run the linter and smoke tests against multiple Ruby versions, so I left those out. Let me know if we should do that too.

Yeah, I agree with you, no need to do that!

Reply:
> btw: I know you from the gorails screencasts (awesome work by the way! I'm really happy to see you around here ❤️)

Thanks! You'll be happy to know that GoRails is going to use Meilisearch now! @cjilbert504 and I have been hacking on it this week upgrading from pg_search. :shipit:

Reply:
Woow, that’s awesome!! I know for sure you will have an awesome experience with us ❤️. 
Let me know if you need anything else 😉